benchmark/solver/solver.cpp

```diff
@@ -351,6 +376,8 @@ void solve_system(const std::string &solver_name,
 
 int main(int argc, char *argv[])
 {
+    // Set the default repetitions = 1.
+    FLAGS_repetitions = 1;
```

Comment:
I think you can set these defaults in the `DEFINE_uint32` macro where you declare the repetitions flag. You can add a short description there as well.

Reply:
What i meant was that you can set the default where you define the flag. If you set it like you do it here, then this value will be used regardless of the parameter that is being passed in from the command line through the `--repetitions=X` flag

Reply:
No, what he did is correct I believe, since you read the value from the user only in `initialize_argument_parsing` which calls gflags. So what he does here is effectively set the default value for this file to be `1`, but if the user passes another value then that will be read later on and will override his settings.

https://github.com/ginkgo-project/ginkgo/blob/da35c7392974ea0eaac921e6651dede9250315ee/benchmark/utils/general.hpp#L98-L121

Reply:
Ah, you want this file to have a default of 1 repetition for all cases ? Okay, then this makes sense.